### PR TITLE
Move the source svg files to src/

### PR DESCRIPTION
**Pull request by orbea**
This is a suggestion, @robloach anyone else  that maintains the assets please review this.

Also see this PR. https://github.com/libretro/RetroArch/pull/5843

My suggestion is to save the source `.svg` files in the `src/` directory and the xmb directory will only contain asset files that should be installed.

Warning, I do not know how much this will disrupt any workflow among those who work on assets. Please show me examples of where this could be problematic and maybe we can come up with a good compromise?

Additionally this will make it easier to maintain the install target all in the RetroArch `qb/` scripts and `Makefile`.

### CODE REVIEW

**Comment by orbea** [comment]
Additionally I noticed several issues in the `Makefile` here.

1. It lacks `$(ASSETS_DIR)`.
2. It arbitrarily changes `$(ASSETS_DIR)/retroarch/assets/` to `share/libretro/assets`. Slackware does not install assets in this location and this would of broken my SlackBuild.
3. It uses possibly non-portably syntax as `$(MAKE)`?
4. It `make` inside a `Makefile`.

File: Makefile [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
-
-install:
-	mkdir -p $(DESTDIR)$(INSTALLDIR)
-	cp -ar * $(DESTDIR)$(INSTALLDIR)
-	rm -rf $(DESTDIR)$(INSTALLDIR)/Makefile \
-		$(DESTDIR)$(INSTALLDIR)/configure \
-		$(DESTDIR)$(INSTALLDIR)/xmb/monochrome/src \
-		$(DESTDIR)$(INSTALLDIR)/xmb/flatui/src \
-		$(DESTDIR)$(INSTALLDIR)/xmb/dot-art/src \
-		$(DESTDIR)$(INSTALLDIR)/xmb/retrosystem/svg
```
Comment (RobLoach) [inline Makefile]: We use this Makefile for Lakka and FlatPak.

**Comment by orbea** [comment]
If it helps you I can spend some time to improving it instead, but would you mind explaining how Lakka and FlatPak use it?

**Comment by RobLoach** [comment]
https://github.com/libretro/Lakka-LibreELEC/blob/Lakka-V2.1-dev/packages/libretro/retroarch-assets/package.mk#L42

**Comment by RobLoach** [comment]
https://github.com/flathub/org.libretro.RetroArch/blob/master/org.libretro.RetroArch.json#L83

**Comment by orbea** [comment]
I see, they install assets as a standalone package. Given this I think improving the `Makefile` here is out of the scope of the goal of this PR and should wait till the next pass.

So I rebased my PR with just moving the `.svg` files to `src/` and updating the `Makefile` accordingly.

Is there any other concerns you can share? What's the workflow for making assets like?

**Comment by orbea** [comment]
I see, they install assets as a standalone package. Given this I think improving the `Makefile` is out of scope for this PR and should wait till the next pass.

So I rebased my PR with just moving the `.svg` files to `src/` and updating the `Makefile` accordingly.

Are there any other concerns you can share? What's the workflow for making assets like?

**Comment by orbea** [comment]
@RobLoach any comments?

**Comment by RobLoach** [comment]
Looks pretty good, simplifies the clean up. Would leave the reviews to @baxysquare and @Alcaro.

**Comment by Alcaro** [comment]
I don't see why I'm relevant. Do whatever.

**Comment by baxysquare** [comment]
To address issue #24 about a year ago, I decided to move my source files over to my own [separate repository](https://github.com/baxysquare/baxy-retroarch-themes).  I presume that the lack of source files has no bearing on this issue for my themes because of that fact.  If that's not the case, let me know and I'd be happy to upload the SVGs once again.

My workflow is different because my primary source files are Adobe Illustrator PDF.  I currently batch create the SVGs using a script created by @alfrix and my PNG files are done using an ImageMagick batch command.  From there I run my PNGs through ImageOptim (#116) and then post the PNGs on my both repositories.

**Comment by orbea** [comment]
I don't think your current workflow interferes here. I just moved the remaining src .svg files to src/ to make it easier for make install. So unless anyone else has anything to add I think this is ready to be merged so that the makefiles can be cleaned up a little bit next?

**Comment by orbea** [comment]
If and when this is merged, I suggest any and all source files be kept in the `src/` directory. That said if they prove to be too large, keeping them in another repo like baxysquare seems fine too.

**Comment by orbea** [comment]
@twinaphex I resolved the conflicts, can you merge this?

**Pull request merged**